aospy/test/test_roundtrip.py

```diff
@@ -150,33 +151,53 @@ def test_vert_int_sigma(self):
                                  dtype_out_time='av',
                                  dtype_out_vert='vert_int',
                                  dtype_in_vert='sigma',
-                                 **self.temp_test_params)
+                                 **self.three_d_test_params)
+        calc = Calc(calc_int)
+        calc.compute()
+
+    def test_sub_daily(self):
+        """Not tested for now in models other than idealized_moist.
+        and idealized_moist_rad."""
+        if not getattr(self, 'sub_daily_test_params', False):
```

Comment:
Hmm...could you maybe get to it through one of the dunder attributes, like `__class__`?  At the same time, because these derive from `unittest.TestCase` and use `setUp`, inheritance stuff can be pretty whacky.

Ignore for now -- surely we'll revamp the test suite sooner or later, and if this is still around at that point we'll revisit.